src/components/ctf/ctf.component.tsx

```diff
@@ -8,9 +8,8 @@ const CTFComponent = () => {
         <div className="container">
             <section id="main-content" className="about">
                 <div className="hero-container">
-                    <h1>Intervarsity Capture the Flag 2025</h1>
-                    <p>Put your white hat hacking skills to the test!</p>
-                    <img src="/assets/img/ctf_poster.png" alt="Italian Trulli"></img>
+                    <h1>CompSoc Intervarsity Capture the Flag - 2025</h1>
+                    <img src="/assets/img/ctf/High_Res_CompSoc_CTF_Poster.jpg" alt="Floor Debugging"></img>
```

Comment:
The poster takes up all the screen and the page contents are hidden, maybe do something like in the following image & code suggestion?
![image](https://github.com/user-attachments/assets/685f86d4-a67d-46ea-9a78-89a7b145538d)

```suggestion
                    <img src="/assets/img/ctf/High_Res_CompSoc_CTF_Poster.jpg" alt="Floor Debugging" style={{maxHeight: "80vh"}}></img>